src/eko/evolution_operator/__init__.py

```diff
@@ -45,6 +46,72 @@ def select_singlet_element(ker, mode):
     return ker[k, l]
 
 
+spec = [
+    ("is_singlet", nb.boolean),
+    ("is_log", nb.boolean),
+    ("logx", nb.float64),
+    ("u", nb.float64),
+]
+
+
+@nb.experimental.jitclass(spec)
```

Comment:
Experimental sounds a bit worrying, but I guess it might be fine (if you maintain for a while...).

Reply:
I thought that as well.

I'm not sure that's the case, but we might want to convert the string in a couple of integers (they have to be integers and not boolean, because in the QED case the coupled sector will be larger than 2x2, namely 4x4).

Reply:
Even a `u1` should be fine, if available.